Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -453,7 +525,13 @@ Extra: newpass=12345 (java.lang.String)
 
 #### Remediation
 
--- TODO [Describe the best practices that developers should follow to prevent this issue.] --
+If not strictly required, be sure that your IPC does not have the `android:exported="true"` value in the `AndroidManifest.xml` file nor an `<intent-filter>`, as otherwise this allows all other apps on Android to communicate and invoke it.
```

Comment:
I meant the "IPC component element" both for "IPC" and for "it". 
I have already refer to them on line 300:
"within the component element (`<activity>`, `<service>`, `<receiver>`)."

It should be clearer now, thanks :)